packages/ui/src/overview/components/DeadlineList/ElectionList.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { ListItem } from '@/common/components/List'
+import { formatDuration } from '@/common/components/statistics'
+import { DurationValue } from '@/common/components/typography/DurationValue'
+import { Subscription } from '@/common/components/typography/Subscription'
+import { useElectionRemainingPeriod } from '@/council/hooks/useElectionRemainingPeriod'
+import { useElectionStage } from '@/council/hooks/useElectionStage'
+
+import {
+  ContentWrapper,
+  ElementWrapper,
+  StyledBadge,
+  StyledClosedButton,
+  StyledLink,
+  StyledText,
+  StyledTriangle,
+  TimeWrapper,
+  TopElementsWrapper,
+} from './styles'
+
+export interface ElectionListProps {
+  electionId: string
+  title: string
+  duration?: string
+}
+
+export const ElectionList = ({ electionId, title }: ElectionListProps) => {
+  const { t } = useTranslation('overview')
+  const { stage: electionStage } = useElectionStage()
+  const remainingPeriod = useElectionRemainingPeriod(electionStage)
+  const timeRemaining = formatDuration(remainingPeriod?.toNumber() || 0)
+  const [hideElement, setHideElement] = useState(false)
+  const remainingCalculation = () => {
+    const dayChecker = timeRemaining[0][1] === 'd'
+    const dayNumberChecker = timeRemaining[0][0]
+    if (
+      (dayNumberChecker < 1 && dayChecker) ||
+      (timeRemaining[0][1] !== 'd' && timeRemaining[0][1] !== 'w') ||
+      dayNumberChecker === 0
+    ) {
+      return '1day'
+    }
+    if (dayNumberChecker > 1 && dayNumberChecker < 3 && dayChecker) {
+      return '3day'
+    }
+  }
+
+  const closeDeadline = () => {
```

Comment:
Instead of useState you can use `useToggle` and then this function can be removed